tests/timepart_constraints.test/runtest.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
This script does not need to be checked in.

Reply:
removed